### ISSUE TITLE
Configuration option for size of small windows

### ISSUE THREAD

**Issue by anoother** [ianyh/Amethyst#1557]
**Is your feature request related to a problem? Please describe.**
Currently there is a boolean option 'float-small-windows', but it affects windows 500px square which by modern standards is *very* small

**Describe the solution you'd like**
Add a configuration option 'small-window-size' which allows you to specify the square-pixel area of windows you'd like Amethyst to consider 'small' and this float.

**Describe alternatives you've considered**
Alternaitvely, modify the existing option to accept *either* a boolean (for which the current default can continue to apply) or an integer, in which case 'true' will be implied and the integer will be taken as the square pixel size.

**Comment by DanCates** [comment]
Agreed. This would be fantastic to have more control over this feature. I struggle with Get Info windows all the time in Finder as well as many other "small" windows that are not small enough for the inherent criteria.

**Comment by ToshihitoKon** [comment]
On my MacBook (3024×1890), the usual Google auth popup is 500×599.
I hope this size can be shown as a floating window.

https://github.com/ianyh/Amethyst/blob/edb5ed5528c34acae30e4f1556da1831a7ca5117/Amethyst/Model/Window.swift#L272%E2%96%AA
